### ISSUE TITLE
high response region question

### ISSUE THREAD

**Issue by ouceduxzk** [Charleo85/DeepCar#1]
In the RA-CNN paper, the author claimed that they pretrained apn model by selecting squares with the highest response value in the last conv layer. Did you do the same in your impl ? 

`def high_response_region(network, inputs):
//     cnn = network.cnn
//     features = cnn(inputs).view(inputs.size[0], 512, 7, 7)
//     square_norm = (features * features).sum(dim = 1)
    unit = torch.Tensor([0, 0, 1])
    return torch.stack([unit] * inputs.size(0))`

**Comment by JUSTDODoDo** [comment]
hello,I am very interested in your work on RA-CNN Implemented  by pytorch, but as a newbie, how do I prepare to train the dataset, I have already got the entire dataset of Compcars, you have not explained clearly in Readme. Training, can you tell the details, thank you!
